examples/migrations/2222-task.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE distributors (
```

Comment:
Evitaria problemas de tabela já existir no banco.
```suggestion
CREATE TABLE IF NOT EXISTS distributors (
```